manage_certs.py

```diff
@@ -45,6 +45,7 @@ def parse_command_line(args):
     parser.add_argument("--consul-token")
     parser.add_argument("--deploy-hook", default='/usr/local/sbin/deploy_cert.sh')
     parser.add_argument("--dns-delay", type=int, default=120)
+    parser.add_argument("--failure_alert_receiver", default=None)
```

Comment:
@ahmedbilal, use `-` in flags instead of `_`. Also, better to call it `--failure-alert-email`.